src/lib.rs

```diff
@@ -197,10 +220,22 @@ impl View for FlexiLoggerView {
         // The longest line sets the width
         let w = logs
             .iter()
-            .map(|msg| msg.width())
+            .map(|msg|
+                msg.spans().map(|x|
+                    // if the log message contains more than one line,
+                    // only the longest line should be considered
+                    // (definitely not the total content.len())
+                    x.content.split('\n').map(|x| x.len()).max().unwrap()
```

Comment:
Should still use `x.width()` instead of `x.len()` to account for multi-byte characters ;).

Reply:
Yes, this is also broken in my previous implementation. Thanks for the hint!